nodejs/nodejs-cloud-run-hello-world/cloudbuild.yaml

```diff
@@ -0,0 +1,76 @@
+steps:
+
+- id: 'Code Style'
+  name: 'node:12-slim'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    npm install
+    npm run lint
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}', '.']
+
+- id: 'Push Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}']
```

Comment:
In .net template I changed it to ${_SERVICE_TAG}${COMMIT_SHA} (removed : )

The reason: when invoking cloud build locally using cloudbuild.yaml, it errors due to special characters. $COMMIT_SHA is null when invoking locally. I suggest we do that in the rest of the templates so they are easily invokable both in CI pipelines and locally.

Reply:
My solution was to keep a "normal" tag approach and document how to run the pipeline:

```
gcloud builds submit . --config cloudbuild.yaml --substitutions COMMIT_SHA=manual
```

I don't think this is much more arcane to the typical developer and manually running the tests won't be something developers typically do with the code.

Reply:
I like that. Will change the dotnet one accordingly.